src/main/java/com/twilio/ivrrecording/servlet/agent/CallServlet.java

```diff
@@ -1,56 +1,56 @@
+import java.io.IOException;
+import java.util.Objects;
 
 import javax.servlet.ServletException;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-import java.io.IOException;
-import java.util.Objects;
-
-public class CallServlet extends WebAppServlet {
-
-    public void doPost(HttpServletRequest request, HttpServletResponse response)
-            throws ServletException, IOException {
-
 
-        String dialCallStatus = request.getParameter("DialCallStatus");
-        String agentId = request.getParameter("agentId");
-
-        if (Objects.equals(dialCallStatus, "completed")) {
-            respondContent(response, "");
-            return;
-        }
-
-        TwiMLResponse twiMLResponse = new TwiMLResponse();
-
-        Say say1 = new Say("It appears that no agent is available. " +
-                "Please leave a message after the beep");
-
-        say1.setLanguage("en-GB");
-        say1.setVoice("alice");
-
-        Record record = new Record();
-        record.setMaxLength(20);
-        record.setAction("/agents/hangup");
-        record.setTranscribeCallback(String.format("/records/create?agentId=%s", agentId));
-
-        Say say2 = new Say("No record received. Goodbye");
-        say2.setLanguage("en-GB");
-        say2.setVoice("alice");
+import com.twilio.ivrrecording.servlet.WebAppServlet;
+import com.twilio.twiml.Hangup;
+import com.twilio.twiml.Record;
+import com.twilio.twiml.Say;
+import com.twilio.twiml.VoiceResponse;
 
-        Hangup hangup = new Hangup();
+public class CallServlet extends WebAppServlet {
 
-        try {
-            twiMLResponse.append(say1);
-            twiMLResponse.append(record);
-            twiMLResponse.append(say2);
-            twiMLResponse.append(hangup);
-        } catch (TwiMLException e) {
-            e.printStackTrace();
-        }
+  public void doPost(HttpServletRequest request, HttpServletResponse response)
+      throws ServletException, IOException {
+    String dialCallStatus = request.getParameter("DialCallStatus");
+    String agentId = request.getParameter("agentId");
 
-        respondTwiML(response, twiMLResponse);
+    if (Objects.equals(dialCallStatus, "completed")) {
+      respondContent(response, "");
+      return;
     }
+
+    Say say1 = new Say.Builder(
+        "It appears that no agent is available. " + "Please leave a message after the beep")
+            .language(Say.Language.EN_GB)
+            .voice(Say.Voice.ALICE)
+            .build();
+
+    Record record = new Record.Builder()
+        .maxLength(20)
+        .action("/agents/hangup")
+        .transcribeCallback(String.format("/records/create?agentId=%s", agentId))
+        .build();
+
+    Say say2 = new Say.Builder("No record received. Goodbye")
+        .language(Say.Language.EN_GB)
+        .voice(Say.Voice.ALICE)
+        .build();
+
+    Hangup hangup = new Hangup();
+
+    VoiceResponse voiceResponse = new VoiceResponse.Builder()
```

Comment:
Although this is a perfectly valid approach, I think we can increase the readability of this piece of code by removing variables like `say1`, `say2` and so on, something like this:

```java
VoiceResponse voiceResponse = new VoiceResponse.Builder()
        .say(new Say.Builder(
                "It appears that no agent is available. " + "Please leave a message after the beep")
                .language(Say.Language.EN_GB)
                .voice(Say.Voice.ALICE)
                .build())
        .record(new Record.Builder()
                .maxLength(20)
                .action("/agents/hangup")
                .transcribeCallback(String.format("/records/create?agentId=%s", agentId))
                .build())
        .say(new Say.Builder("No record received. Goodbye")
                .language(Say.Language.EN_GB)
                .voice(Say.Voice.ALICE)
                .build())
        .hangup(new Hangup())
        .build();
```

What do you think?